camayoc/tests/qpc/yupana/test_connect.py

```diff
@@ -0,0 +1,41 @@
+"""Starting tests for configs"""
+
+import requests
+
+from camayoc.tests.qpc.yupana.utils import (
+    yupana_configs,
+    get_app_url,
+    post_file,
+    get_x_rh_identity,
+)
+
+
+EXPECTED_CONFIGS = ("app_name", "openshift_project", "openshift_app_domain", "api_path")
+
+
+def test_config():
+    """test load configurations"""
+    config = yupana_configs()
+    assert config != []
+    assert sorted(set(config["yupana-app"].keys())) == sorted(EXPECTED_CONFIGS)
+
+
+def test_connect():
+    """Test the connection to the desired cluster."""
+    request = requests.get(get_app_url() + "status/")
+    assert request.status_code == 200
```

Comment:
Include `request.text` to the assertion, so we can guess why the test failed. I usually do `assert request.status_code == 200, request.text`, to not keep the assertion without a message.

I expect to use the variable name `response` or `resp` or `r` as the result when calling `requests.*()` , due the nature of HTTP: you do a request and you get the answer or response :)

Example:
```python
>>> resp = requests.get('http://redhat.com/123')                           
>>>resp.status_code                                                       
404

>>> assert resp.status_code == 200, resp.text                              

AssertionError: <!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
        "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
	<title>redhat.com | The World's Open Source Leader</title>
...
```

Reply:
In this case I think is useful to provide the message since only the pytest information won't show all the information to help debug, good call

Reply:
Ah, I meant to have `response` as the var name and wrote the wrong thing for that test :laughing: .

Also, good call on returning the `response.text`, I added it.